### ISSUE TITLE
node >= 10.18 is necessary ?

### ISSUE THREAD

**Issue by 0x0006e** [wwayne/react-tooltip#627]
### Problem

- Node version in my CI runner was 10.15.3.
- When CI install react-tooltip will throw `error react-tooltip@4.2.8: The engine "node" is incompatible with this module. Expected version ">=10.18". Got "10.15.3"`

### So
- does node >= 10.18 is really necessary ? 
- react-tooltip package depends on node >=10.18 ?

package.json
```js
"engines": {
    "node": ">=10.18",
    "npm": ">=6.13"
  },
```

**Comment by 0x0006e** [comment]
fixed 4.2.9

**Comment by roggervalf** [comment]
#615
